xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-profile/xwiki-platform-user-profile-test/xwiki-platform-user-profile-test-docker/src/test/it/org/xwiki/user/test/ui/UserProfileIT.java

```diff
@@ -71,7 +71,7 @@
 
     private static final String USER_BLOGFEED = "http://xwiki.org/feed";
 
-    private static final String WYSIWYG_EDITOR = "Wysiwyg";
+    private static final String WYSIWYG_EDITOR = "WYSIWYG";
```

Comment:
This is actually a piece of code dedicated to test the interface: it will looks for the String "Wysiwyg", changing it is more likely to break some tests since it won't find that String anymore.

It's more likely that the changes should take place in the templates, in xwiki-platform-web or in xwiki-platform-flamingo-skin-resources. It's also possible that some of them are in wiki pages in XML files.

Reply:
> 
> 
> This is actually a piece of code dedicated to test the interface: it will looks for the String "Wysiwyg", changing it is more likely to break some tests since it won't find that String anymore.
> 
> It's more likely that the changes should take place in the templates, in xwiki-platform-web or in xwiki-platform-flamingo-skin-resources. It's also possible that some of them are in wiki pages in XML files.

So what kind of changes should I make? CSS text-transform?

Reply:
No here most of the work is only to locate where those strings are used in the UI and where they are defined in the code. And, instead of using each time a new version of "WYSIWYG" string, the idea is to always rely on the same translation key: that way we ensure consistency over the whole interface, and it might be change in another language by translating it.